### PR TITLE
Update stable/influxdb/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Delete dulicated word.
 Line 72: a Persistent Volume volume

### CODE REVIEW

**Comment by timstoop** [comment]
The reason that it's failing is that some of the requirements of the Chart.yaml have changed recently and this Chart is missing an appVersion key. You could add it and simply set it to the version of the image that's default. That would move this forward and fix it for future commits to this Chart as well.

**Comment by AdamDang** [comment]
@timstoop Thanks! I've add the appVersion key.

File: stable/influxdb/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: influxdb
-version: 0.8.3
+version: 0.8.4
+appVersion: 0.8.4
```
Comment (sameersbn) [inline stable/influxdb/Chart.yaml]: looking at https://github.com/AdamDang/charts/blob/5afd5234e898ab4d0a79e28016d6bceb554b8bca/stable/influxdb/values.yaml#L5, the appVersion should probably be `1.4`

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/influxdb/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 name: influxdb
-version: 0.8.3
+version: 0.8.4
+appVersion: 1.4-alpine
```
Comment (sameersbn) [inline stable/influxdb/Chart.yaml]: you dont need to add the `-alpine` part. it should just be the version part

**Review by sameersbn — CHANGES_REQUESTED** [review]

**Review by sameersbn — APPROVED** [review]

**Pull request merged**